wx/py/filling.py

```diff
@@ -106,10 +106,7 @@ def OnItemActivated(self, event):
 
     def objHasChildren(self, obj):
         """Return true if object has children."""
-        if self.objGetChildren(obj):
-            return True
-        else:
-            return False
+        return type(obj) not in COMMONTYPES
```

Comment:
Looking at `objGetChildren`, do I need to add the following?
```python
return ('BTrees' in six.text_type(otype) and hasattr(obj, 'keys') or ...
```
I don't know what `Btrees` type is.